### PR TITLE
Add template and bootstrap command for Codespaces dev container

### PR DESCRIPTION
**Pull request by shadyvb**
Ref humanmade/altis-dev-tools#346

### CODE REVIEW

File: inc/command/class-command.php [hunk 1]
```diff
@@ -445,6 +451,70 @@ protected function codecept( InputInterface $input, OutputInterface $output ) {
...
+
+			return 1;
+		}
+
+		if ( ! in_array( $subsubcommand, $configs, true ) ) {
+			$output->writeln( sprintf( '<error>Could not find the target configuration set generator for "%s".</error>', $subsubcommand ) );
+			return 1;
+		}
+
+		call_user_func( [ $this, 'bootstrap_' . $subsubcommand ], $input, $output );
```
Comment (roborourke) [inline inc/command/class-command.php:481]: You could also do something like:
```suggestion
		$method = "bootstrap_{$subsubcommand}";
		$this->$method( $input, $output );
```

**Review by roborourke — APPROVED** [review]
Looks good, got one suggestion inline but makes no difference, just a note really. Otherwise my only suggestion would be to provide something like the choice interface like we have for composer server logs but this is cool as is

File: inc/command/class-command.php (hunk 1 above)
Comment (mikelittle) [reply inc/command/class-command.php:481]: By the way, the curly braces are usually not needed in a interpolated string unless followed by a character that could be considered part of the variable.
so: `$method = "bootstrap_$subsubcommand";`

Comment (shadyvb) [reply inc/command/class-command.php:481]: @roborourke I'd prefer using `call_user_func` as it seems more readable to me, would you agree ? is the requested change more consistent with how other bits of the code looks like ?

@mikelittle re the braces, I'd opt for consistency when it's low effort like this as a general preference. eg: if we have two instances of interpolated strings, I'd want them to look consistent, eg: `something_{$var}` and `something_{$var}else`. Also looks specially readable if you don't have syntax highlighting on.

Comment (roborourke) [reply inc/command/class-command.php:481]: Yeah that’s fine to use call_user_func, I don’t think there’s any precedent in this module for it

File: templates/.devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.192.0/containers/php
+{
+	"name": "PHP",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			"PHP_VERSION": "8",
```
Comment (mikelittle) [inline templates/.devcontainer/devcontainer.json:8]: This doesn't match the format of the "8.0" default in the DockerFile, is the format ok as just "8"?

**Review by mikelittle — COMMENTED** [review]
Just one question around version format.

File: templates/.devcontainer/devcontainer.json (hunk 2 above)
Comment (shadyvb) [reply templates/.devcontainer/devcontainer.json:8]: Just seen this after merge, sorry. So yes, you can still use 8, but it does make sense to make 8.0 the default here as well for consistency. Will open a new PR for it. Good shout!

**Pull request merged**